### ISSUE TITLE
Unsent fix handling in SAP Sail InSight tracking app incorrect

### ISSUE THREAD

**Issue by aksajhfduwafe** [aksajhfduwafe/bugzilla-to-github-issues-test#3629]
| Field | Value |
|---|---|
| **Reporter** | @axeluhl |
| **Assignee** | @axeluhl |
| **Product** | Sailing Race Analytics |
| **Component** | Smartphone Tracking |
| **Version** | unspecified |
| **Hardware** | All |
| **OS** | All |
| **Severity** | normal |
| **Priority** | P2 |
| **CC (unmapped)** | Bennet Brunsen (`brunsen@ubilabs.net`), Steffen Wagner (`mail.sap@steffenwagner.com`), Samuel Oey (`oey@ubilabs.net`) |
| **Created** | 2016-04-26T10:46:57Z |
| **Original Bug** | [Bug 3629](https://bugzilla.sapsailing.com/bugzilla/show_bug.cgi?id=3629) |

---

When I start tracking, then put the phone into Flight Mode, the "Unsent Fixes" count remains at "1." Only after going back out of Flight Mode into online mode does the app show "160" unsent fixes. To make matters worse, those fixes are then sent one-by-one, wasting major amounts of bandwidth and causing significant delays. The /v1/gps_fixes endpoint clearly allows for sending a whole set of fixes with one call which is what the app should be doing. Create a separate task if you will, but I guess both aspects should be solved more or less in the same place in the app's code.

**Comment by aksajhfduwafe** [comment]
**Steffen Wagner (`mail.sap@steffenwagner.com`)** commented:

I would guess the same applies for iOS as well?
